crates/sui-framework/sources/prover.move

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::prover {
+
+    use sui::object;
+
+    const OWNED: u64 = 1;
+    const SHARED: u64 = 2;
+    const IMMUTABLE: u64 = 3;
+
+    #[verify_only]
+    /// Information about which object contains a given object field (stored at the field object's
+    /// address).
+    struct DynamicFieldContainment has key {
```

Comment:
Is this used anywhere right now?

Reply:
That's an artifact of me thinking about the reverse mapping which we probably won't need. Will remove!